dir.props

```diff
@@ -89,26 +89,607 @@
          /t:BatchRestorePackages /p:OverridePackageSource=C:\coreclr\bin\Product\Windows_NT.x64.Debug\.nuget\pkg
     -->
     <DnuSourceList Include="$(OverridePackageSource)" />
-    <!-- PackagesDrops is passed in via the commandline "/p:PackagesDrops=[drop location]" -->
-    <DnuSourceList Condition="'$(BuildTestsAgainstPackages)' == 'true'" Include="$(PackagesDrops)" />
     <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
     <!-- Including buildtools to pull in TestSuite packages and repackaged xunit dependencies-->
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-buildtools/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fdotnet.myget.org/F/dotnet-core/api/v3/index.json" />
     <DnuSourceList Include="https:%2F%2Fwww.nuget.org/api/v2/" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- PackagesDrops is passed in via the commandline "/p:PackagesDrops=[drop location]" -->
+    <DnuSourceList Condition="'$(BuildTestsAgainstPackages)' == 'true'" Include="$(PackageInstallPath)" />
+  </ItemGroup>
+
   <!-- This is the directory where we dynamically generate project.json's for our test build package dependencies -->
   <PropertyGroup Condition="'$(BuildTestsAgainstPackages)' == 'true'">
     <BuildTestsAgainstPackagesIdentityRegex>$(CoreFxVersionsIdentityRegex)</BuildTestsAgainstPackagesIdentityRegex>
     <SkipVerifyPackageVersions>true</SkipVerifyPackageVersions>
   </PropertyGroup>
+  
+    
+    <!--Temporary workaround for these runtime specific packages that aren't built locally-->
+  <PropertyGroup>
+    <TempVersion>$(PackageVersion)-$(CoreFxExpectedPrerelease)</TempVersion>
+    
+    <!-- defaults for BuildTestsAgainstPackages -->
+    <!-- PackageInstallPath is where the locally built packages are installed -->
+    <PackageInstallPath>$(BinDir)localpackages</PackageInstallPath>
+    
+  </PropertyGroup>
+  <ItemGroup>
+    <_InjectProjectReferenceDependency Include="runtime.native.System.Security.Cryptography">
```

Comment:
@ericstj If you want to change them back to runtime references in master that's fine, mainly it was to prove out the build/test flow before doing it in 1.1.0, since you said it was changing anyways. I think it was also making the work @ellismg is doing harder as direct dependencies.